tethys_portal/settings.py

```diff
@@ -407,8 +407,8 @@
 STATICFILES_DIRS = [
     BASE_DIR / "static",
 ]
-if has_module(bokehjsdir):
-    STATICFILES_DIRS.append(bokehjsdir())
+if has_module(bokehjs_path):
+    STATICFILES_DIRS.append(bokehjs_path())
```

Comment:
```suggestion
if has_module(bokeh_settings):
    try:
        bokeh_js_dir = bokeh_settings.bokehjs_path()
    except AttributeError:
        # support bokeh versions < 3.4
        bokeh_js_dir = bokeh_settings.bokehjsdir()
    STATICFILES_DIRS.append(bokeh_js_dir)
```

Reply:
This suggested change not only calls `bokehjs_path` on the `bokeh_settings` object as is the intended usage (see comment above), but it also maintains backward compatibility so it will still work with Bokeh versions < 3.4.